mitosheet/src/mito/components/taskpanes/Merge/MergeTaskpane.tsx

```diff
@@ -233,7 +317,7 @@ const MergeTaskpane = (props: MergeTaskpaneProps): JSX.Element => {
                     {params.how !== MergeType.UNIQUE_IN_LEFT && 
                         <MultiToggleColumns
                             sheetData={sheetDataTwo}
-                            selectedColumnIDs={params.selected_column_ids_two.concat(mergeKeyColumnIDsTwo)}
```

Comment:
Unless we handle these on the backend, don't we need to add these?

Reply:
I was confused about this - my understanding of why it works is that:
* We list all columns in the `MultiToggle`
* We select and disable any merge keys by default
* The merge keys are necessarily in the selected columns

Reply:
I removed this because it was showing duplicate columns in the warning messages